### PR TITLE
Correct install.php

### PR DESCRIPTION
**Pull request by oliverpool**
Hi,

thank you for this project, that I'm trying to install (on PHP5.6).

I think that there are some issues with the `dbDriver` selection during the installation:
- the `Illuminate\Console\Command` already manages the default argument, so the [`askQuestion`](https://github.com/satrun77/tinyissue/blob/master/app/Console/Commands/Install.php#L398) can be simplified.
- the choices should not be prefixed with `pdo_`
- I have an empty db password: it should be the default

### CODE REVIEW

**Comment by satrun77** [comment]
Thanks for the pull request.

**Comment by satrun77** [comment]
Unfortunately, the patch failed. The password still required and the database driver failing with the following error "Unsupported driver [l]" :(

You can choose between:
1) You fix the patch, and I will test it again.
2) I can modify your patch and merge it.

Thanks for your work

**Comment by oliverpool** [comment]
I'll let you modify it (I now use another fork of tinyissue: https://github.com/pixeline/bugs because it support tags update)

File: app/Console/Commands/Install.php [hunk 1]
```diff
@@ -248,9 +248,14 @@ protected function formatTableCells(array $cells, $color)
      */
     protected function getValidDbDrivers()
     {
-        return array_keys(array_filter($this->dbDrivers, function ($item) {
-            return $item === true;
-        }));
+        return array_map(
+                function ($pdo_name){
+                    return substr($pdo_name, strlen('pod_'));
```
Comment (oliverpool) [inline app/Console/Commands/Install.php]: Strictly speaking, it should be `pdo` here (but it doesn't change the length ;-)

**Pull request merged**